xbmc/addons/kodi-dev-kit/include/kodi/addon-instance/pvr/Recordings.h

```diff
@@ -424,6 +426,25 @@ class PVRRecording : public CStructHdl<PVRRecording, PVR_RECORDING>
   /// @brief To get with @ref SetChannelType changed values
   PVR_RECORDING_CHANNEL_TYPE GetChannelType() const { return m_cStructure->channelType; }
 
+  /// @brief **optional**\n
+  /// Channel type.
+  ///
+  /// Set to @ref PVR_RECORDING_MEDIA_TYPE_UNKNOWN if the type cannot be
```

Comment:
The new enum has no such value. And the whole implementation is missing support for this case.

Reply:
Fix. API states that media type should always be known to the client and the default value will be PVR_RECORDING_MEDIA_TYPE_RECORDING

Reply:
Perfect.